templates/tftmpl/condition.go

```diff
@@ -40,24 +44,61 @@ type Condition interface {
 
 // ServicesCondition handles appending templating for the services run condition
 // This is the default run condition
-type ServicesCondition struct{}
+type ServicesCondition struct {
+	Regexp string
+}
 
 func (c ServicesCondition) SourceIncludesVariable() bool {
 	return false
 }
 
+func (c ServicesCondition) ServicesAppended() bool {
+	return c.Regexp != ""
+}
+
 func (c ServicesCondition) appendModuleAttribute(body *hclwrite.Body) {}
 
-func (c ServicesCondition) appendTemplate(io.Writer) error {
-	// no-op: services condition currently requires no additional condition
-	// templating. it relies on the monitoring template as the run condition
+func (c ServicesCondition) appendTemplate(w io.Writer) error {
+	q := c.hcatQuery()
+	if c.Regexp != "" {
+		_, err := fmt.Fprintf(w, serviceRegexTmpl, q)
+		if err != nil {
+			log.Printf("[WARN] (templates.tftmpl) unable to write service condition template")
```

Comment:
I am wondering if there's more context for the log level `WARN`.

This log line corresponds to an error, do you know if this error would terminate CTS during startup? If so, changing the log level to `ERR` would be more representative.

Reply:
Not sure, let me try to force an error and see the behavior!

Reply:
It does terminate CTS on the error, both here and with catalog services condition. I'll update both places to have `ERR` instead of `WARN`.